components/library/goffice/Makefile

```diff
@@ -49,12 +50,27 @@ CONFIGURE_ENV+= PYTHON=$(PYTHON)
 COMPONENT_INSTALL_ENV+= GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
 
 # common targets
-build:		$(BUILD_32)
+build:		$(BUILD_32_and_64)
 
-install:	$(INSTALL_32)
+install:	$(INSTALL_32_and_64)
 
-test:		$(NO_TESTS)
+unexport SHELLOPTS
+
+test:		$(TEST_32_and_64)
 
 BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
```

Comment:
Haven't spotted this - remove BUILD_PKG_DEPENDENCIES and include $(WS_TOP)/make-rules/depend.mk lines - this is legacy , we don't use them.

Reply:
Also change $(WS_TOP)/make-rules to $(WS_MAKE_RULES)